stable/insights-agent/values.yaml

```diff
@@ -119,13 +119,13 @@ goldilocks:
         memory: 545Mi
   dashboard:
     enabled: false
-  resources:
```

Comment:
in goldilocks chart there is no such thing as `values.resources` there is only `values.controller.resources` and `values.dashboard.resources`